src/software/simulated_tests/multiple_robots_enter_region.py

```diff
@@ -0,0 +1,109 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class RobotEntersRegion(Validation):
+
+    """Checks if a Robot enters any of the provided regions."""
+
+    def __init__(self, regions=None, count=1):
+        self.regions = regions if regions else []
+        self.count = count
+
+    def get_validation_status(self, world) -> ValidationStatus:
+        """Checks if _any_ robot enters the provided regions
+
+        :param world: The world msg to validate
+        :returns: FAILING until a robot enters any of the regions
+                  PASSING when a robot enters
+        """
+        for region in self.regions:
+            for robot in world.friendly_team.team_robots:
+                if tbots.contains(
+                    region, tbots.createPoint(robot.current_state.global_position)
+                ):
+                    return ValidationStatus.PASSING
+
+        return ValidationStatus.FAILING
+
+    def get_validation_geometry(self, world) -> ValidationGeometry:
+        """
+        (override) shows regions to enter
+        """
+        return create_validation_geometry(self.regions)
+
+    def __repr__(self):
+        return "Check for robot in regions " + ",".join(
+            repr(region) for region in self.regions
+        )
+
+
+(
+    RobotEventuallyEntersRegion,
+    RobotEventuallyExitsRegion,
+    RobotAlwaysStaysInRegion,
+    RobotNeverEntersRegion,
+) = create_validation_types(RobotEntersRegion)
+
+
+class NumberOfRobotsEntersRegion(Validation):
```

Comment:
this class seems the same as NumberOfRobotsEntersRegion in robot_enters_region.py but with 1 region instead of many. we can probably just use the old class with a 1 element list, so this seems a bit redundant

Reply:
deleted `multiple_robots_enters_region.py` but this same issue could be raised with `robot_enters_region.py` - will make this change